src/vision/status_quo/niklaus_wants_to_share_knowledge.md

```diff
@@ -0,0 +1,38 @@
+# 😱 Status quo stories: Niklaus Wants to Share Knowledge
+
+
+## 🚧 Warning: Draft status 🚧
+
+This is a draft "status quo" story submitted as part of the brainstorming period. It is derived from real-life experiences of actual Rust users and is meant to reflect some of the challenges that Async Rust programmers face today. 
+
+If you would like to expand on this story, or adjust the answers to the FAQ, feel free to open a PR making edits (but keep in mind that, as they reflect peoples' experiences, status quo stories [cannot be wrong], only inaccurate). Alternatively, you may wish to [add your own status quo story][htvsq]!
+
+## The story
+
+Niklaus, who sometimes goes by the pen name "Starol Klichols", has authored some long-form documentation about Rust that people have found helpful. One could even go so far as to call this documentation a ["book"][trpl].
+
+Niklaus has typically minimized the use of crates in documentation like this as much as possible. Niklaus has limited time to dedicate to keeping the documentation up to date, and given the speed at which the ecosystem sometimes evolves, it's hard to keep up when crates are involved. Also, Niklaus would like to avoid limiting the readership of the documentation to the users of a particular crate only, and would like to avoid any accusations of favoritism.
+
+But Niklaus would really really like to document async to avoid disappointing [people like Barbara]!
+
+Niklaus was excited about [the RFC proposing that `block_on` be added to the stdlib][block-on-rfc], but understands that everyone is busy. However, if that RFC came to be, that would solve Niklaus' problems and make it more likely that `async` would be added to the documentation. Niklaus would really like to include `async` in a big update to the documentation that would likely happen with the Rust 2021 Edition. No pressure.
```

Comment:
Hmmm, this is kinda sneaking in some shiny future @carols10cents -- that's cheating!!

Reply:
Still, it's also a factual experience I suppose.

Reply:
Perhaps we could just generalize this to "features that are only practically useable with a third-party" crate and just use `block_on` as an example that comes up. I don't want people to get the idea that stabilizing `block_on` would fix this issue in general even if it fixed Starol Klichols issue 😉

Reply:
```suggestion
Niklaus was excited about [the RFC proposing that `block_on` be added to the stdlib][block-on-rfc], because it seemed like that would solve Niklaus' problems. Niklaus would really like to include `async` in a big update to the documentation. No pressure.
```

I'm not sure if that's what you mean @rylev -- i couldn't quite envision what you had in mind. i tried to pare this back to what niklaus felt and wanted.

Reply:
I think that’s certainly closer. We should certainly merge this, but I will keep an eye out to make sure this story doesn’t get interpreted too narrowly. I think the moral of this story is very widely applicable.